test/ocsp/helper/helper.go

```diff
@@ -16,12 +16,70 @@ import (
 	"golang.org/x/crypto/ocsp"
 )
 
-var method = flag.String("method", "GET", "Method to use for fetching OCSP")
-var urlOverride = flag.String("url", "", "URL of OCSP responder to override")
-var hostOverride = flag.String("host", "", "Host header to override in HTTP request")
-var tooSoon = flag.Int("too-soon", 76, "If NextUpdate is fewer than this many hours in future, warn.")
-var ignoreExpiredCerts = flag.Bool("ignore-expired-certs", false, "If a cert is expired, don't bother requesting OCSP.")
-var expectStatus = flag.Int("expect-status", 0, "Expect response to have this numeric status (0=good, 1=revoked)")
+var (
+	method             = flag.String("method", "GET", "Method to use for fetching OCSP")
+	urlOverride        = flag.String("url", "", "URL of OCSP responder to override")
+	hostOverride       = flag.String("host", "", "Host header to override in HTTP request")
+	tooSoon            = flag.Int("too-soon", 76, "If NextUpdate is fewer than this many hours in future, warn.")
+	ignoreExpiredCerts = flag.Bool("ignore-expired-certs", false, "If a cert is expired, don't bother requesting OCSP.")
+	expectStatus       = flag.Int("expect-status", -1, "Expect response to have this numeric status (0=Good, 1=Revoked, 2=Unknown)")
```

Comment:
Let's keep the default value here as `0` until we have confirmation from SRE that all uses have been updated to pass this flag explicitly.

Reply:
Nice, makes sense. Done.